dolphinscheduler-ui/src/js/conf/home/index.js

```diff
@@ -17,6 +17,7 @@
 
 // The Vue build version to load with the `import` command
 // (runtime-only or standalone) has been set in webpack.base.conf with an alias.
+import 'babel-polyfill'
 import Vue from 'vue'
 import App from './App'
 import router from './router'
```

Comment:
Keep spaces

Reply:
It's just ok, I don't know why it turns red